src/extensions/plg_osmap_com_content/com_content.php

```diff
@@ -537,6 +547,13 @@ public static function includeCategoryContent($osmap, $parent, $catid, &$params,
                 $node->catslug = $item->catid;
                 $node->link    = ContentHelperRoute::getArticleRoute($node->slug, $node->catslug);
 
+                if (strpos($node->link,'Itemid=')===false) {
+                    $node->itemid = $itemid;
+                    $node->link .= '&Itemid='.$parent->id;
+                } else {
+                    $node->itemid = $itemid;
+					$node->link = preg_replace('/Itemid=([0-9]+)/','Itemid='.$parent->id,$node->link);
```

Comment:
We removed this since it was changing the SEF URLs for some links in the sitemap